apps/transport/lib/jobs/geo_data/irve_to_geo_data.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Transport.Jobs.IRVEToGeoData do
+  @moduledoc """
+  Job in charge of taking the charge stations stored in the Base nationale des Infrastructures de Recharge pour Véhicules Électriques and storing the result in the `geo_data` table.
+  """
+
+  use Oban.Worker, max_attempts: 3
+  import Ecto.Query
+  require Logger
+
+  @etalab_organization_id "534fff75a3a7292c64a77de4"
+  @resource_datagouv_id "8d9398ae-3037-48b2-be19-412c24561fbb"
+
+  def perform(%{}) do
```

Comment:
```suggestion
  @impl Oban.Worker
  def perform(%Oban.Job{}) do
```
https://hexdocs.pm/oban/Oban.html#module-defining-workers